src/index.html

```diff
@@ -15,8 +15,52 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="true"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="nav-container">
+        <a
+          href="#"
+          class="logo"
+        >
+          <img
+            src="./images/logo.png"
+            alt="Moyo logo"
+          />
+        </a>
+        <nav>
+          <ul>
+            <li><a href="https://moyo.com/home">Home</a></li>
+            <li><a href="https://moyo.com/home">Products</a></li>
+            <li><a href="https://moyo.com/home">About us</a></li>
+            <li>
+              <a
+                href="https://moyo.com/home"
+                class="is-active"
+                data-qa="hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li>
+              <a href="https://moyo.com/home">Contact Us</a>
```

Comment:
All navigation links point to the same URL 'https://moyo.com/home'. This might be an error unless it's intentional. Consider updating the href attributes to point to the correct pages for 'Products', 'About us', 'Laptops & Computers', and 'Contact Us'.